bin/configure-tower-projects.py

```diff
@@ -528,16 +528,16 @@ def create_credentials(self) -> int:
                 assert cred["deleted"] is None
                 return cred["id"]
         # Otherwise, create a new credentials entry for the project
-        secret_arn = self.stack["TowerForgeServiceUserAccessKeySecretArn"]
+        secret_arn = self.stack["TowerRoleArn"]
```

Comment:
So now `TowerRoleArn` serves the purpose of both `TowerForgeServiceUserAccessKeySecretArn` and `TowerForgeServiceRoleArn`?

Reply:
Yup. This is my understanding after reviewing Seqera instruction. We would be able to use IAM role arn rather than IAM user's secrets generated for the user to authenticate AWS.